libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway/Base.hs

```diff
@@ -675,35 +673,6 @@ instance
           SNothing -> pure (Agda.MkTxId 0, 0)
         pure (committee, agdaLastId)
 
-instance
-  ( SpecTranslate ctx (PParamsHKD Identity era)
-  , Inject ctx (EnactState era)
-  , Inject ctx (Proposals era)
-  , EraPParams era
-  , SpecRep (PParamsHKD Identity era) ~ Agda.PParams
-  ) =>
-  SpecTranslate ctx (GovEnv era)
-  where
-  type SpecRep (GovEnv era) = Agda.GovEnv
-
-  toSpecRep GovEnv {..} = do
-    props <- askCtx @(Proposals era)
-    enactState <- askCtx @(EnactState era)
-    let
-      protVer = gePParams ^. ppProtocolVersionL
-      -- TODO express this in agdaConstraints rather than doing it during
-      -- translation
-      enactStatePV =
-        enactState
-          & ensProtVerL .~ protVer
-          & ensPrevGovActionIdsL .~ toPrevGovActionIds (props ^. pRootsL)
```

Comment:
@Lucsanszky you must have noticed that this has been removed and added only to the general `translateInputs` function, while earlier, overriding the `ensPrevGovActionIds` was being done twice: once here and once in `translateInputs`, which was superfluous.

Reply:
Good catch!